lib/rs_api/rs_player_experience.rb

```diff
@@ -20,9 +20,13 @@ def initialize(username)
 
     def display
       table_data if table.rows.empty?
-      puts table
+      puts table if display?
     rescue PlayerNotFound => e
-      puts "#{e} | '#{@username}' does not exist."
+      puts "#{e} | '#{@username}' does not exist." if display?
+    end
+
+    def display?
+      RsApi.load_config['display_output']
     end
```

Comment:
Could this be turned into a method elsewhere?

If not, then in the future when `PlayerCompare` and `PlayerExperience` inherent from a base class, move method there

Reply:
Leaving this for future work. This could be a helper method also